src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js

```diff
@@ -659,6 +648,32 @@ const getCurrentPublishedAdvisories = async (cmsData, setCmsData) => {
               isClearable
             />
           </div>
+          <div className="col-xl-3 col-md-4 col-sm-12">
+            <FormControlLabel style={{ marginLeft: '5px' }} control={
+              <Checkbox
+                checked={showArchived}
+                onChange={(e) => {
+                  let showArchived = e ? e.target.checked : false;
+                  let arr = [...filters.filter((o) => !(o.type === 'page' && o.filterName === 'archived'))];
+                  setFilters([...arr, { type: 'page', filterName: 'archived', filterValue: showArchived }]);
+                  toggleArchivedAdvisories(showArchived)
+                }}
+                inputProps={{ "aria-label": "archived" }}
+              />
+            } label={
+              <>
+                <small style={{ paddingRight: '6px' }}>
```

Comment:
This padding should be removed, because PR https://github.com/bcgov/bcparks.ca/pull/769 adds a margin-left to the HelpIcon